crates/sui-replay/src/lib.rs

```diff
@@ -343,6 +363,42 @@ pub async fn execute_replay_command(
             // TODO: clean this up
             Some((0u64, 0u64))
         }
+        ReplayToolCommand::ProfileTransaction {
+            tx_digest,
+            show_effects,
+            diag,
+            executor_version_override,
+            protocol_version_override,
+            profile_output_filepath_override,
+        } => {
+            let tx_digest = TransactionDigest::from_str(&tx_digest)?;
+            info!("Executing tx: {}", tx_digest);
+            let sandbox_state = LocalExec::replay_with_network_config(
+                rpc_url,
+                cfg_path.map(|p| p.to_str().unwrap().to_string()),
```

Comment:
I'm not a huge fan of the `unwrap` in here -- is there a reason why this needs to be a string, vs a path?

Reply:
this was copied from the existing several instances, I'll update all in a follow-up